package.json

```diff
@@ -10,16 +10,21 @@
     "private::lint.pylint-dash": "PYLINTRC=${PYLINTRC:=.pylintrc37} && pylint dash setup.py --rcfile=$PYLINTRC",
     "private::lint.pylint-tests": "PYLINTRC=${PYLINTRC:=.pylintrc37} && pylint tests/unit tests/integration -d all --rcfile=$PYLINTRC",
     "private::lint.renderer": "cd dash-renderer && npm run lint",
-    "private::test.setup-nested": "cd \\@plotly/dash-generator-test-component-nested && npm ci && npm run build && pip install -e .",
-    "private::test.setup-standard": "cd \\@plotly/dash-generator-test-component-standard && npm ci && npm run build && pip install -e .",
+    "private::test.setup-nested": "cd \\@plotly/dash-generator-test-component-nested && npm ci && npm run build",
+    "private::test.setup-standard": "cd \\@plotly/dash-generator-test-component-standard && npm ci && npm run build",
+    "private::test.py.deploy-nested": "npm run private::test.setup-nested && cd \\@plotly/dash-generator-test-component-nested && pip install -e .",
+    "private::test.py.deploy-standard": "npm run private::test.setup-standard && cd \\@plotly/dash-generator-test-component-standard && pip install -e .",
+    "private::test.R.deploy-nested": "npm run private::test.setup-nested && cd \\@plotly/dash-generator-test-component-nested && sudo R CMD INSTALL .",
+    "private::test.R.deploy-standard": "npm run private::test.setup-standard && cd \\@plotly/dash-generator-test-component-standard && sudo R CMD INSTALL .",
     "private::test.unit-dash": "PYTHONPATH=~/dash/tests/assets pytest tests/unit",
     "private::test.unit-renderer": "cd dash-renderer && npm run test",
     "private::test.integration-dash": "TESTFILES=$(circleci tests glob \"tests/integration/**/test_*.py\" | circleci tests split --split-by=timings) && pytest --headless --nopercyfinalize --junitxml=test-reports/junit_intg.xml ${TESTFILES}",
     "format": "run-s private::format.*",
     "initialize": "run-s private::initialize.*",
     "lint": "run-s private::lint.*",
-    "setup-tests": "run-s private::test.setup-*",
-    "test.integration": "run-s setup-tests private::test.integration-*",
+    "setup-tests.py": "run-s private::test.py.deploy-*",
+    "setup-tests.R": "run-s private::test.R.deploy-*",
```

Comment:
@Marc-Andre-Rivet Will this work within `3.7-stretch-node-browsers`? 

I would have thought you'd either need to layer in an R image or install the binary in order to run R within the Python base image.